### PR TITLE
JAMES-2856 Add QuotaRoot as part of mailbox details over webadmin

### PR DESCRIPTION
**Pull request by billyfrost418**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8] [ERROR] Failures: 
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8] [ERROR]   QuotaRootTest.quotaRootWithDomainShouldBeWellDeSerialized:47 
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8] Expecting:
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8]  <QuotaRoot{value=bob@domain.tld, domain=Optional.empty}>
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8] to be equal to:
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8]  <QuotaRoot{value=bob@domain.tld, domain=Optional[Domain : domain.tld]}>
[63ed332f24fd466ebd33bc00faac8d6ecace0cb8] but was not.
```
seems to be related to the PR. Please check

**Comment by duongtq** [comment]
Read it

**Comment by trantienduchn** [comment]
```
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] [ERROR] Errors: 
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] [ERROR]   SpamAssassinContract.expungingSpamMessageShouldNotImpactSpamAssassinState:493 » ConditionTimeout
```
test this please

**Comment by trantienduchn** [comment]
```
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] Location: /org/apache/james/imap/scripts/ValidateDeployment.test:499
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] LastClientMsg: c SEARCH FLAGGED
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] Expected: '\* SEARCH 1 2 3 4'
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] Actual   : '* SEARCH'
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] 	at org.apache.james.mpt.imapmailbox.external.james.DeploymentValidation.validateDeployment(DeploymentValidation.java:81)
```
test this please

**Comment by Arsnael** [comment]
```
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] [ERROR] Errors: 
[5a2ebd7dbf212b28c1e7ae45afbfb4aea6fc833a] [ERROR]   MailsShouldBeWellReceived.mailsShouldBeWellReceived:63 » ConditionTimeout Cond...
```
test this please

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -350,18 +350,29 @@ Response codes:
...
 
 ```
-curl -XGET http://ip:port/users/usernameToBeUsed/mailboxes/mailboxNameToBeCreated
+curl -XGET http://ip:port/users/usernameToBeUsed/mailboxes/mailboxNameToBeUsed
 ```
 
+will return :
+
+```
+{"quotaRoot":"#private&usernameToBeUsed"}
```
Comment (mbaechler) [inline src/site/markdown/server/manage-webadmin.md]: Do you think the value make sense to a user?

**Pull request closed without merging**